src/components/sponsor-tiers/sponsor-tiers.tsx

```diff
@@ -166,6 +163,20 @@ export const SponsorTiers = () => {
         ]}
       />
 
+      <SponsorTier
+        title="Platinum X"
+        totalSlots={3}
```

Comment:
```suggestion
        totalSlots={4}
```

Reply:
@artcz mentioned in the sponsor doc that probably we should scale this down to 3, as we'd offer 8 Platinum in total this year as opposed to 4 last year. Having 3 means we have an upper limit of 6.

Reply:
I would keep it at 4, though, the Platinum X doesn't include a booth, and we don't know how this package would sell.